tests/test.py

```diff
@@ -1437,8 +1475,8 @@ def tearDown(self):
         if not self.EXTERNAL_ACRA and hasattr(self, 'acra'):
             procs.append(self.acra)
         stop_process(procs)
-        send_signal_by_process_name('acra-server', signal.SIGTERM)
-        send_signal_by_process_name('acra-connector', signal.SIGTERM)
+        send_signal_by_process_name('acra-server', signal.SIGKILL)
+        send_signal_by_process_name('acra-connector', signal.SIGKILL)
```

Comment:
Why do we need to use `SIGKILL` instead of `SIGTERM`? Is there any problem with process termination dusring test?

Reply:
siterm + sigkill after timeout sent via `stop_process` function above that uses pid's that was returned after expected fork from our code. `send_signal_by_process_name` use pidof to finally kill any zombie processes that may be forked after unexpected sighup signal.  because after sighup the processes that we forked from tests create own fork with new pid that we don't know from test environment and terminate itself.

in general cases `send_signal_by_process_name` will do nothing because all processes was terminated correctly before.

Reply:
Thanks for the explanation.